core-bundle/contao/themes/flexible/styles/main.css

```
@@ -2477,7 +2484,9 @@ ul.sgallery li {
 	}
 }
 
+.tl_sectionwizard td:first-child,
 .tl_sectionwizard td:last-child {
```

```suggestion
.tl_modulewizard td:is(:first-child, :last-child) {
```
Just a nit, ignore if not needed :)